ppb/__init__.py

```diff
@@ -1,13 +1,31 @@
+import logging
+from typing import Callable
+
 from ppb.vector import Vector
 from ppb.engine import GameEngine
-
 from ppb.scenes import BaseScene
 from ppb.sprites import BaseSprite
 
-import logging
 
-def run(starting_scene=BaseScene, *, log_level=logging.WARNING, **kwargs):
+def run(setup: Callable[[BaseScene], None], *, log_level=logging.WARNING):
```

Comment:
I actually disagree with the exclusion of the scene class/object from this signature. But I also tend to override scenes a lot.

Still, many of my demos have a scene class with event handlers of some sort.